packages/disploy/src/structs/PartialGuildMember.ts

```diff
@@ -1,8 +1,12 @@
-import type { APIGuildMember, APIInteractionDataResolvedGuildMember, Snowflake } from 'discord-api-types/v10';
+import type { APIGuildMember, APIInteractionDataResolvedGuildMember, APIUser, Snowflake } from 'discord-api-types/v10';
 import type { App } from '../client';
 import { Base } from './Base';
 
 export class PartialGuildMember extends Base {
+	/**
+	 * The id of the member.
+	 */
+	public id!: Snowflake | null;
```

Comment:
Unessarary, while yes most of the code base uses the `!:` syntax, we're migrating away from it and no new code should be using it. Also, the id will never be null from the API, but since you're using Partial, TypeScript assumes anything could be missing.
```suggestion
	public id: Snowflake;
```